backend/routes/authRoutes.js

```diff
@@ -55,4 +57,10 @@ router.post('/reset-password', [
     check('newPassword', 'Password must be at least 6 characters').isLength({ min: 6 }) // Validate new password
 ], resetPassword);
 
+// Get Profile Route
+router.get('/profile', authMiddleware, getProfile);
+
+// Update Profile Route
+router.put('/profile', authMiddleware, updateProfile);
```

Comment:
Having a userRoute so that endpoints can be in the form of `.../user/profile, .../user/:id/listings` seems more conventional. I believe /auth usually refers to endpoints for register, login, and activities in that flow.

Reply:
This is a great idea and something we should discuss. We can either create a new file to separate concerns (`authRoutes` vs. `userRoutes`) or refactor `authRoutes` to differentiate between `authRouter` and `userRouter`.